docs/specification/draft/client/sampling.md

```diff
@@ -5,10 +5,10 @@ weight: 40
 ---
 
 {{< callout type="info" >}}
-**Protocol Revision**: draft
+**Protocol Revision**: {{< param protocolRevision >}}
```

Comment:
I think we want this page to always say "draft," rather than a specific date, to indicate that it's not codified yet.